index.js

```diff
@@ -24,16 +27,26 @@ var Dropzone = React.createClass({
     activeStyle: React.PropTypes.object,
     className: React.PropTypes.string,
     activeClassName: React.PropTypes.string,
+    rejectClassName: React.PropTypes.string,
 
     disableClick: React.PropTypes.bool,
     multiple: React.PropTypes.bool,
+    accept: React.PropTypes.string,
+  },
+
+  allFilesAccepted: function(files) {
+    return files.every(file => accept(file, this.props.accept))
```

Comment:
BTW I just realized this will not work because of arrow function. Do you mind using ES6 in your project? I could submit another PR with the complete solution.